packages/Jms/src/Rector/Property/JmsInjectAnnotationRector.php

```diff
@@ -0,0 +1,128 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Jms\Rector\Property;
+
+use Nette\Utils\Strings;
+use PhpParser\Node;
+use PhpParser\Node\Stmt\Property;
+use PHPStan\PhpDocParser\Ast\PhpDoc\PhpDocTagNode;
+use Rector\Bridge\Contract\AnalyzedApplicationContainerInterface;
+use Rector\NodeTypeResolver\Node\Attribute;
+use Rector\NodeTypeResolver\PhpDoc\NodeAnalyzer\DocBlockAnalyzer;
+use Rector\Rector\AbstractRector;
+use Rector\RectorDefinition\CodeSample;
+use Rector\RectorDefinition\RectorDefinition;
+
+final class JmsInjectAnnotationRector extends AbstractRector
+{
+    /**
+     * @var string
+     */
+    private const INJECT_ANNOTATION = 'JMS\DiExtraBundle\Annotation\Inject';
+
+    /**
+     * @var DocBlockAnalyzer
+     */
+    private $docBlockAnalyzer;
+
+    /**
+     * @var AnalyzedApplicationContainerInterface
+     */
+    private $analyzedApplicationContainer;
+
+    public function __construct(
+        DocBlockAnalyzer $docBlockAnalyzer,
+        AnalyzedApplicationContainerInterface $analyzedApplicationContainer
+    ) {
+        $this->docBlockAnalyzer = $docBlockAnalyzer;
+        $this->analyzedApplicationContainer = $analyzedApplicationContainer;
+    }
+
+    public function getDefinition(): RectorDefinition
+    {
+        return new RectorDefinition(
+            'Changes properties with @JMS\DiExtraBundle\Annotation\Inject to constructor injection',
+            [
+                new CodeSample(
+                    <<<'CODE_SAMPLE'
+use JMS\DiExtraBundle\Annotation as DI;
+
+class SomeController
+{
+    /**
+     * @DI\Inject("entity.manager")
+     */
+    private $entityManager;
```

Comment:
public

Reply:
In previous examples you've send it was private. What is the correct then?

Any docs link I could add to Rector to explain?

Reply:
Is it this one? 
https://jmsyst.com/bundles/JMSDiExtraBundle/master/annotations#inject

Reply:
yes, i meant to say, they can be **private**

Reply:
Kk, now are ale accepted. Thanks to you and your feedback. 
Before it was only limited to public.